### PR TITLE
Polish translation for LAMBS Danger

### PR DESCRIPTION
**Pull request by MiszczuZPolski**
I saw that you needed help with the translation into Polish, so I decided that I would help.

### CODE REVIEW

File: addons/danger/stringtable.xml [hunk 1]
```diff
@@ -4,275 +4,346 @@
...
         </Key>
         <Key ID="STR_Lambs_Danger_Module_SetRadio_DisplayName">
             <English>Configure Long-range Radio</English>
             <German>Langstrecken-Funk einstellen</German>
+            <Polish>Konfiguruj długie radio</Polish>
         </Key>
         <Key ID="STR_Lambs_Danger_Module_SetRadio_SettingName">
             <English>Toggle boosted communication range on unit</English>
             <German>Verstärkte Kommunikationsreichweite für diese Einheit umschalten</German>
+            <Polish>Przełącza zwiększony zasięg komunikacji dla tego urządzenia</Polish>
```
Comment (veteran29) [inline addons/danger/stringtable.xml:27]:
```suggestion
            <Polish>Przełącza zwiększony zasięg komunikacji dla tej jednostki</Polish>
```

File: addons/danger/stringtable.xml [hunk 2]
```diff
@@ -4,275 +4,346 @@
...
         </Key>
         <Key ID="STR_Lambs_Danger_Module_SetRadio_SettingToolTip">
             <English>Unit with radio toggled have boosted communications range when sharing information\nThis effect is also achieved by equipping the unit with a Vanilla Radio Backpack or TFAR-mod enabled radio.</English>
             <German>Einheiten bei denen diese Option aktiviert ist haben eine erhöhte Kommunikationsreichweite.\n Dieser Effekt kann auch erzeugt werden durch das Ausrüsten der Einheit mit einem standard Radio-Rucksack oder einem TFAR-Mod kompatiblen Radio.</German>
+            <Polish>Jednostka z włączonym radiem ma zwiększony zasięg komunikacji podczas przekazywania informacji \n Ten efekt może być również osiągnięty, poprzez wyekwipowanie jednostki w plecak z radiem</Polish>
```
Comment (veteran29) [inline addons/danger/stringtable.xml:32]:
```suggestion
            <Polish>Jednostka z włączonym radiem ma zwiększony zasięg komunikacji podczas przekazywania informacji \n Ten efekt może być również osiągnięty, poprzez wyekwipowanie jednostki w plecak z radiem lub plecak kompatybilny z modem TFAR</Polish>
```

File: addons/danger/stringtable.xml [hunk 3]
```diff
@@ -4,275 +4,346 @@
...
             <German>Maximaler 'knowsAbout' Wert der zwischen Einheiten geteilt wird.\nDas Setzten dieses Wertes größer 1 kann dazu führen, dass entfernte Verbündete in den Gefahrenmodus wechseln.</German>
+            <Polish>Maksymalna wartość „knowAbout” dzielona między jednostki. \n Ustawienie tej wartości większej niż 1 może spowodować, że odlegli sojusznicy przejdą w tryb zagrożenia.</Polish>
         </Key>
         <Key ID="STR_Lambs_Danger_Settings_combatShareRange">
             <English>Range combat mode is shared</English>
+            <Polish>Obszar wspólnej walki</Polish>
         </Key>
         <Key ID="STR_Lambs_Danger_Settings_combatShareRange_ToolTip">
             <English>Friendly groups within range will be set to combat behaviour. Units in combat mode will seek cover and concealment and turn towards nearest known threat.</English>
+            <Polish>Obszar na którym sojusznicze jednostki dołączą do walki. Jednostki w trybie walki będą szukały osłony i kierować się w stronę najbliższego znanego zagrożenia.</Polish>
```
Comment (veteran29) [inline addons/danger/stringtable.xml:287]:
```suggestion
            <Polish>Obszar na którym sojusznicze jednostki dołączą do walki. Jednostki w trybie walki będą szukały osłony i kierowały się w stronę najbliższego znanego zagrożenia.</Polish>
```

File: addons/danger/stringtable.xml [hunk 4]
```diff
@@ -4,275 +4,346 @@
...
         </Key>
         <Key ID="STR_Lambs_Danger_Settings_maxRevealValue">
             <English>Accuracy of shared information</English>
             <German>Genauigkeit der geteilten Informationen</German>
+            <Polish>Dokładność udostępnianych informacji</Polish>
         </Key>
         <Key ID="STR_Lambs_Danger_Settings_maxRevealValue_ToolTip">
             <English>Maximum 'knowsAbout' value shared between groups.\nSetting greater than 1 may cause distant allies to go into full danger mode.</English>
             <German>Maximaler 'knowsAbout' Wert der zwischen Einheiten geteilt wird.\nDas Setzten dieses Wertes größer 1 kann dazu führen, dass entfernte Verbündete in den Gefahrenmodus wechseln.</German>
+            <Polish>Maksymalna wartość „knowAbout” dzielona między jednostki. \n Ustawienie tej wartości większej niż 1 może spowodować, że odlegli sojusznicy przejdą w tryb zagrożenia.</Polish>
```
Comment (veteran29) [inline addons/danger/stringtable.xml:279]:
```suggestion
            <Polish>Maksymalna wartość "knowsAbout" dzielona między jednostki. \n Ustawienie tej wartości większej niż 1 może spowodować, że odlegli sojusznicy przejdą w tryb zagrożenia.</Polish>
```

File: addons/eventhandlers/stringtable.xml [hunk 5]
```diff
@@ -4,22 +4,27 @@
...
         </Key>
         <Key ID="STR_Lambs_Eventhandlers_Settings_EnabledExplosionEH_ToolTip">
             <English>Toggle AI Reactions to nearby explosions</English>
             <German>KI-Reaktionen auf umliegende Explosionen umschalten</German>
+            <Polish>Przełącza reakcję SI na pobliskie wybuchy</Polish>
         </Key>
         <Key ID="STR_Lambs_Eventhandlers_Settings_ExplosionResetTime_DisplayName">
             <English>Explosion Eventhandler reset time</English>
             <German>Explosionsereignis-Handler Reset-Timer</German>
+            <Polish>Licznik czasu resetowania modułu obsługi zdarzenia wybuchu</Polish>
```
Comment (veteran29) [inline addons/eventhandlers/stringtable.xml:22]:
```suggestion
            <Polish>Czas resetowania reakcji na zdarzenie wybuchu</Polish>
```

File: addons/eventhandlers/stringtable.xml [hunk 6]
```diff
@@ -4,22 +4,27 @@
...
         </Key>
         <Key ID="STR_Lambs_Eventhandlers_Settings_ExplosionResetTime_ToolTip">
             <English>Configures explosions reset time.</English>
             <German>Legt den Reset-Timer für den Explosionsereignis-Handler fest.</German>
+            <Polish>Konfiguruje czas resetowania wybuchu</Polish>
```
Comment (veteran29) [inline addons/eventhandlers/stringtable.xml:27]:
```suggestion
            <Polish>Konfiguruje czas resetowania reakcji na zdarzenie wybuchu.</Polish>
```

**Review by veteran29 — CHANGES_REQUESTED** [review]

**Comment by diwako** [comment]
Here is me, pretending to understand all of this. Keep it up :D

**Review by veteran29 — CHANGES_REQUESTED** [review]

File: addons/danger/stringtable.xml [hunk 7]
```diff
@@ -4,275 +4,346 @@
         <Key ID="STR_Lambs_Danger_Module_DisableAI_DisplayName">
             <English>Disable Unit AI</English>
             <German>KI der Einheit ausschalten</German>
+            <Polish>Wyłącz SI dla jednostek</Polish>
```
[the remark this answers is not in the document]
Comment (MiszczuZPolski) [reply addons/danger/stringtable.xml:7]: Ok, no problem

File: addons/main/stringtable.xml [hunk 8]
```diff
@@ -31,66 +34,82 @@
...
         </Key>
         <Key ID="STR_Lambs_Main_Settings_DisableHandGestures_Tooltip">
             <English>Toggles unit gestures and hand signals when reacting to danger or executing planned manoevures</English>
             <German>Deaktiviert Handzeichen von Einheiten beim Reagieren auf Gefahr oder das durchführen von Manövern.</German>
+            <Polish>Wyłącza używanie gestów ręką podczas reagowania na niebezpieczeństwo lub podczas wykonywania manewrów</Polish>
         </Key>
         <Key ID="STR_Lambs_Main_Settings_DisableCallouts">
             <English>Disable unit callouts</English>
             <German>Rufe ausschalten</German>
+            <Polish>Wyłącz połączenia</Polish>
```
Comment (veteran29) [inline addons/main/stringtable.xml:52]:
```suggestion
            <Polish>Wyłącz okrzyki</Polish>
```

File: addons/main/stringtable.xml [hunk 9]
```diff
@@ -31,66 +34,82 @@
...
         </Key>
         <Key ID="STR_Lambs_Main_Settings_DisableCallouts_ToolTip">
             <English>Toggles extra unit callouts based on situations</English>
             <German>Deaktiviert zusätzliche Rufe von Einheiten basierend auf der Sitation.</German>
+            <Polish>Wyłącza dodatkowe wywołania od jednostek w zależności od sytuacji</Polish>
```
Comment (veteran29) [inline addons/main/stringtable.xml:57]:
```suggestion
            <Polish>Wyłącza dodatkowe wołania/okrzyki jednostek w zależności od sytuacji</Polish>
```

File: addons/wp/stringtable.xml [hunk 10]
```diff
@@ -4,441 +4,552 @@
...
         </Key>
         <Key ID="STR_Lambs_WP_Module_TaskCQB_DeleteOnStartUp_DisplayName">
             <English>Remove Dynamic center</English>
             <German>Dynamischen Mittelpunkt entfernen</German>
+            <Polish>Usuń dynamiczne centrum</Polish>
         </Key>
         <Key ID="STR_Lambs_WP_Module_TaskCQB_DeleteOnStartUp_Tooltip">
             <English>With the dynamic center present, it is possible to move the center point of the building search pattern</English>
             <German>Wenn der dynamische Mittelpunkt vorhanden ist, kann der Mittelpunkt für das Gebäudesuchmuster verschoben werden.</German>
+            <Polish>Gdy dynamiczne centrum jest widoczne, jest możliwe, żeby poruszać zasięgiem zadania.</Polish>
```
Comment (veteran29) [inline addons/wp/stringtable.xml:197]:
```suggestion
            <Polish>Gdy dynamiczne centrum jest widoczne, jest możliwe, żeby poruszać centralnym punktem przeszukiwania budynku.</Polish>
```

File: addons/wp/stringtable.xml [hunk 11]
```diff
@@ -4,441 +4,552 @@
...
         </Key>
         <Key ID="STR_Lambs_WP_Module_TaskReset_ModuleDescription">
             <English>This module resets all custom LAMBS scripts running on the unit</English>
             <German>Dieses Modul setzt alle benutzerdefinierten LAMBS Skripte zurück die für diese Einheit laufen.</German>
+            <Polish>Resetuje wszystkie skrypty LAMBS działające na jednostce.</Polish>
         </Key>
         <Key ID="STR_Lambs_WP_Module_TaskCreep_DisplayName">
             <English>Task Creep</English>
             <German>Aufgabe Schleichen</German>
+            <Polish>Task - Zasadzka</Polish>
```
Comment (veteran29) [inline addons/wp/stringtable.xml:325]:
```suggestion
            <Polish>Zadanie - Zasadzka</Polish>
```

File: addons/wp/stringtable.xml [hunk 12]
```diff
@@ -4,441 +4,552 @@
...
         </Key>
         <Key ID="STR_Lambs_WP_Module_TaskPatrol_ModuleDescription">
             <English>Task Patrol makes the unit patrol within the area set by 3DEN attributes.</English>
             <German>Task Patroullie lässt Einheiten innerhalb des Gebietes das durch die Eden Editor Attribute festgelegt wurde, patrouillieren</German>
+            <Polish>Zadanie - Patrol pozwala jednostkom na patrolowanie obszaru.</Polish>
         </Key>
         <Key ID="STR_Lambs_WP_Module_TaskReset_DisplayName">
             <English>Task Reset</English>
             <German>Aufgabe Zurücksetzen</German>
+            <Polish>Task - Reset</Polish>
```
Comment (veteran29) [inline addons/wp/stringtable.xml:315]:
```suggestion
            <Polish>Zadanie - Reset</Polish>
```

File: addons/wp/stringtable.xml [hunk 13]
```diff
@@ -4,441 +4,552 @@
...
             <German>Distanz in der die schleichende Gruppe nach Feinden sucht.\nStandardmäßig visiert diese Modul nur Spieler an.</German>
+            <Polish>Obszar na którym grupa będzie szukać przeciwników. \n Domyślnie, moduł jest stworzony aby używać go przeciwko wrogim graczom.</Polish>
         </Key>
         <Key ID="STR_Lambs_WP_Module_TaskCreep_ModuleDescription">
             <English>Task Creep makes the unit attempt to ambush enemies within the area set by 3DEN attributes. The group will use available cover and keep a low stance to remain undiscovered.&lt;br/&gt;&lt;br/&gt;By default search modules only target hostile players</English>
             <German>Aufgabe Schleichen bringt die Einheiten dazu, in dem durch die Eden Attribute festgelegtem Gebiete, Feinde zu überfallen. Die Gruppe nutzt verfügbare Deckung und nutzt eine geduckte Haltung um unerkannt zu bleiben.&lt;br/&gt;&lt;br/&gt;Standardgemäß zielt das Such-Modul nur auf Spieler.</German>
+            <Polish>Zadanie - Zasadzka pozwala jednostkom przeprowadzenie zasadzki na przeciwników. Grupa będzie używać osłon i trzymać się nisko w celu utrzymania anonimowości.</Polish>
```
Comment (veteran29) [inline addons/wp/stringtable.xml:370]:
```suggestion
            <Polish>Zadanie - Zasadzka pozwala jednostkom przeprowadzenie zasadzki na przeciwników. Grupa będzie używać osłon i trzymać się nisko w celu uniknięcia wykrycia.&lt;br/&gt;&lt;br/&gt;Domyślnie moduł wyszukuje tylko wrogich graczy.</Polish>
```

File: addons/wp/stringtable.xml [hunk 14]
```diff
@@ -4,441 +4,552 @@
...
         </Key>
         <Key ID="STR_Lambs_WP_Module_TaskCreep_Radius_DisplayName">
             <English>Radius</English>
             <German>Radius</German>
+            <Polish>Promień</Polish>
         </Key>
         <Key ID="STR_Lambs_WP_Module_TaskCreep_Radius_ToolTip">
             <English>Distance creeping group will search for enemies.\nBy default this module only targets enemy players</English>
             <German>Distanz in der die schleichende Gruppe nach Feinden sucht.\nStandardmäßig visiert diese Modul nur Spieler an.</German>
+            <Polish>Obszar na którym grupa będzie szukać przeciwników. \n Domyślnie, moduł jest stworzony aby używać go przeciwko wrogim graczom.</Polish>
```
Comment (veteran29) [inline addons/wp/stringtable.xml:365]:
```suggestion
            <Polish>Obszar na którym grupa będzie szukać przeciwników. \n Domyślnie moduł wyszukuje tylko wrogich graczy.</Polish>
```

File: addons/wp/stringtable.xml [hunk 15]
```diff
@@ -4,441 +4,552 @@
...
             <German>Aufgabe Jagen</German>
+            <Polish>Zadanie - Polowanie</Polish>
         </Key>
         <Key ID="STR_Lambs_WP_Module_TaskHunt_ModuleDescription">
             <English>Task Hunt makes the unit patrol the area within the area set by 3DEN attributes. The group will move casually in a search pattern gradually spiraling closer to the target. The unit will enable flashlights if available.&lt;br/&gt;&lt;br/&gt;By default search modules only target hostile players.</English>
             <German>Aufgabe Jagen bringt die Einheiten dazu, in dem durch die Eden Attribute festgelegtem Gebiet, zu patrouilieren. Die Gruppe nähert sich spiralförmig und gelassen an das Ziel an. Die Einheit wird eine Taschenlampe verwenden, wenn diese verfügbar ist.&lt;br/&gt;&lt;br/&gt;Standardgemäß zielt das Such-Modul nur auf Spieler.</German>
+            <Polish>Zadanie - Polowanie sprawia, że jednostki patrolują obszar określony przez moduł. Grupa podchodzi do celu spiralnie i spokojnie. Jednostka użyje latarek jeśli będzie to możliwe. &lt;br/&gt; &lt;br/&gt; Domyślnie, moduł jest stworzony aby używać go przeciwko wrogim graczom.</Polish>
```
Comment (veteran29) [inline addons/wp/stringtable.xml:380]:
```suggestion
            <Polish>Zadanie - Polowanie sprawia, że jednostki patrolują obszar określony przez moduł. Grupa podchodzi do celu spiralnie i spokojnie. Jednostka użyje latarek jeśli będzie to możliwe.&lt;br/&gt; &lt;br/&gt;Domyślnie moduł wyszukuje tylko wrogich graczy.</Polish>
```

File: addons/wp/stringtable.xml [hunk 16]
```diff
@@ -4,441 +4,552 @@
...
         </Key>
         <Key ID="STR_Lambs_WP_Module_TaskHunt_CycleTime_DisplayName">
             <English>Script interval</English>
             <German>Skript-Interval</German>
+            <Polish>Interwał skryptu</Polish>
         </Key>
         <Key ID="STR_Lambs_WP_Module_TaskHunt_CycleTime_ToolTip">
             <English>The cycle time for the script in seconds. Higher numbers can be used to make the hunting group less accurate.\nDefault 70 seconds</English>
             <German>Zykluszeit für das Skript in Sekunden. Ein höherer Wert macht die jagende Gruppe weniger genau.\nStandardgemäß 70 Sekunden</German>
+            <Polish>Czas cyklu skryptu w sekundach. Wyższy czas sprawi, że jednostki będą mniej celne. \n Domyślnie 70 sekund</Polish>
```
Comment (veteran29) [inline addons/wp/stringtable.xml:410]:
```suggestion
            <Polish>Czas cyklu skryptu w sekundach. Wyższy czas sprawi, że jednostki będą mniej celne.\nDomyślnie 70 sekund</Polish>
```

File: addons/wp/stringtable.xml [hunk 17]
```diff
@@ -4,441 +4,552 @@
...
         </Key>
         <Key ID="STR_Lambs_WP_Module_TaskHunt_Radius_DisplayName">
             <English>Radius</English>
             <German>Radius</German>
+            <Polish>Promień</Polish>
         </Key>
         <Key ID="STR_Lambs_WP_Module_TaskHunt_Radius_ToolTip">
             <English>Distance hunting group will search for enemies.\nBy default this module only targets enemy players</English>
             <German>Die Distanz die die jagende Gruppe nach Feinden sucht.\nStandardgemäß zielt das Such-Modul nur auf Spieler.</German>
+            <Polish>Obszar na którym grupa będzie szukać przeciwników. \n Domyślnie, moduł jest stworzony aby używać go przeciwko wrogim graczom.</Polish>
```
Comment (veteran29) [inline addons/wp/stringtable.xml:420]:
```suggestion
            <Polish>Obszar na którym grupa będzie szukać przeciwników.\nDomyślnie moduł wyszukuje tylko wrogich graczy.</Polish>
```

File: addons/wp/stringtable.xml [hunk 18]
```diff
@@ -4,441 +4,552 @@
...
         </Key>
         <Key ID="STR_Lambs_WP_Module_TaskRush_ModuleDescription">
             <English>Task Rush makes the unit attempt to run aggressively towards enemies within the area set by 3DEN attributes. The unit will move extremely aggressively, often ignoring cover and suppression.&lt;br/&gt;&lt;br/&gt;By default search modules only target hostile players. Disable unit FSM for a truly terrifying enemy.</English>
             <German>Aufgabe Hetzen lässt die Einheit aggressiv auf Feinde in dem durch die Eden Attribute festgelegtem Gebiet losstürmen. Die Einheit wird sich aggressiv bewegen und meistens Deckung und Niederhaltung ignorieren.&lt;br/&gt;&lt;br/&gt;Standardgemäß zielt das Such-Modul nur auf Spieler. Schalten Sie die Einheiten-FSM ab, um einen wirklich furchteinflößenden Feind zu erhalten.</German>
+            <Polish>Zadanie - Szturm sprawia, że jednostki agresywnie szturmują naprzeciw wrogom. Jednostki będą poruszać się bardzo agresywnie, często ignorując osłonę i ogień zaporowy.&lt;br/&gt;&lt;br/&gt;Domyślnie, moduł jest stworzony aby używać go przeciwko wrogim graczom. Wyłącz FSM dla jednostek, dla prawdziwie przerażających przeciwników.</Polish>
```
Comment (veteran29) [inline addons/wp/stringtable.xml:430]:
```suggestion
            <Polish>Zadanie - Szturm sprawia, że jednostki agresywnie szturmują naprzeciw wrogom. Jednostki będą poruszać się bardzo agresywnie, często ignorując osłonę i ogień zaporowy.&lt;br/&gt;&lt;br/&gt;Domyślnie moduł wyszukuje tylko wrogich graczy. Wyłącz FSM dla jednostek, dla prawdziwie przerażających przeciwników.</Polish>
```

File: addons/wp/stringtable.xml [hunk 19]
```diff
@@ -4,441 +4,552 @@
...
         </Key>
         <Key ID="STR_Lambs_WP_Module_TaskRush_CycleTime_DisplayName">
             <English>Script interval</English>
             <German>Skript-Interval</German>
+            <Polish>Interwał skryptu</Polish>
         </Key>
         <Key ID="STR_Lambs_WP_Module_TaskRush_CycleTime_ToolTip">
             <English>The cycle time for the script in seconds. Higher numbers can be used to make rushers less accurate.\nDefault 4 seconds</English>
             <German>Zykluszeit für das Skript in Sekunden. Ein höherer Wert macht die hetzende Gruppe weniger genau.\nStandardgemäß 4 Sekunden</German>
+            <Polish>Czas cyklu skryptu w sekundach. Wyższy czas sprawi, że jednostki będą mniej celne. \n Domyślnie 4 sekund</Polish>
```
Comment (veteran29) [inline addons/wp/stringtable.xml:460]:
```suggestion
            <Polish>Czas cyklu skryptu w sekundach. Wyższy czas sprawi, że jednostki będą mniej celne.\nDomyślnie 4 sekund</Polish>
```

File: addons/wp/stringtable.xml [hunk 20]
```diff
@@ -4,441 +4,552 @@
...
         </Key>
         <Key ID="STR_Lambs_WP_Groups_DisplayName">
             <English>Groups</English>
             <German>Gruppen</German>
+            <Polish>Grupy</Polish>
         </Key>
         <Key ID="STR_Lambs_WP_Groups_ToolTip">
             <English>Select which unit script applies to.\nList is sorted by distance</English>
             <German>Legt fest, auf welche Einheit das Skript angewendet wird.\nDie List ist nach der Entfernung sortiert.</German>
+            <Polish>Określa na jaką jednostkę ma zadziałać skrypt. \n Lista jest sortowana po dystansie.</Polish>
```
Comment (veteran29) [inline addons/wp/stringtable.xml:495]:
```suggestion
            <Polish>Określa na jaką jednostkę ma zadziałać skrypt.\nLista jest sortowana po dystansie.</Polish>
```

**Review by veteran29 — CHANGES_REQUESTED** [review]

**Comment by jokoho48** [comment]
@veteran29 @MiszczuZPolski status?

File: addons/wp/stringtable.xml [hunk 21]
```diff
@@ -4,441 +4,552 @@
...
         </Key>
         <Key ID="STR_Lambs_WP_Module_TaskArtillery_SkipCheckrounds_DisplayName">
             <English>Skip adjusting rounds</English>
             <German>Anpassen der Schüsse überspringen</German>
+            <Polish>Pomiń strzały korygujące</Polish>
         </Key>
         <Key ID="STR_Lambs_WP_Module_TaskArtillery_SkipCheckrounds_Tooltip">
             <English>Check this to disable initial rounds used by the fire controller to adjust rounds on target.\nSkipping this will make the barrage immediately hit on target</English>
             <German>Aktiviere diese Option um ein Kalibrieren der Schüsse zu deaktivieren.\nWenn diese Option aktiviert trifft der Beschuss das Ziel sofort.</German>
+            <Polish>Włącz tę opcję, aby nie wystrzelić strzałów korygujących, używanych przez kontrolera do skorygowania ostrzału na punkt</Polish>
```
Comment (veteran29) [inline addons/wp/stringtable.xml:67]:
```suggestion
            <Polish>Włącz tę opcję, aby nie wystrzelić strzałów korygujących, używanych przez kontrolera do skorygowania ostrzału na punkt.\nWłączenie tej opcji spowoduje natychmiastowy ostrzał na wyznaczony cel.</Polish>
```

File: addons/wp/stringtable.xml [hunk 22]
```diff
@@ -4,441 +4,552 @@
...
         </Key>
         <Key ID="STR_Lambs_WP_Module_TaskAssault_DeleteOnStartup_DisplayName">
             <English>Remove module on Start-Up</English>
             <German>Entferne das Modul beim Start</German>
+            <Polish>Usuń moduł podczas uruchamiania</Polish>
         </Key>
         <Key ID="STR_Lambs_WP_Module_TaskAssault_DeleteOnStartup_Tooltip">
             <English>Deletes module on startup.\nIf unchecked it is possible to move the module to change destination dynamically</English>
             <German>Entfernt das Modul beim Start.\n Wenn nicht aktiviert ist es möglich das Module zu verschieben um den Zielort dynamisch anzupassen.</German>
+            <Polish>Usuwa moduł podczas uruchamiania. \nJeśli opcja nie jest aktywowana, można przemieścić moduł w celu dynamicznego dostosowywania lokalizacji docelowej.</Polish>
```
Comment (veteran29) [inline addons/wp/stringtable.xml:122]:
```suggestion
            <Polish>Usuwa moduł podczas uruchamiania.\nJeśli opcja nie jest aktywowana, można przemieścić moduł w celu dynamicznego dostosowywania lokalizacji docelowej.</Polish>
```

File: addons/wp/stringtable.xml [hunk 23]
```diff
@@ -4,441 +4,552 @@
...
         </Key>
         <Key ID="STR_Lambs_WP_Module_TaskCQB_CycleTime_DisplayName">
             <English>Script interval</English>
             <German>Skript-Interval</German>
+            <Polish>Interwał skryptu</Polish>
         </Key>
         <Key ID="STR_Lambs_WP_Module_TaskCQB_CycleTime_Tooltip">
             <English>The cycle time for the script in seconds. Higher numbers make units search buildings more carefully.\nDefault 21 seconds</English>
             <German>Die Zykluszeit des Skripts in Sekunden. Eine höherer Wert sorgt dafür das Gebäude vorsichtiger von Einheiten durchsucht werden.\nStandardwert ist 21 Sekunden.</German>
+            <Polish>Czas cyklu skryptu w sekundach. Wyższy czas sprawi, że jednostki będą przeszukiwać budynki bardziej ostrożnie. \nDomyślnie 21 sekund</Polish>
```
Comment (veteran29) [inline addons/wp/stringtable.xml:187]:
```suggestion
            <Polish>Czas cyklu skryptu w sekundach. Wyższy czas sprawi, że jednostki będą przeszukiwać budynki bardziej ostrożnie.\nDomyślnie 21 sekund</Polish>
```

**Review by veteran29 — CHANGES_REQUESTED** [review]

**Review by veteran29 — APPROVED** [review]

**Pull request merged**